InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -153,12 +161,55 @@ void GenericHandler::showCommandPrompt(Visualization::Item* commandReceiver, QSt
 {
 	if (commandPrompt_ && commandPrompt_->commandReceiver() == commandReceiver)
 	{
-		commandPrompt_->showPrompt(initialCommandText);
+        commandPrompt_->showPrompt(initialCommandText);
```

Comment:
Please fix the whitespace here and everywhere else. Use only TABs for indentation.